src/types/project.types.ts

```diff
@@ -39,7 +39,7 @@ export type ProjectOrganization = Omit<Organization, 'partOf' | 'hasPart' | 'acr
 
 export type ProjectStatus = 'ACTIVE' | 'CONCLUDED' | 'NOTSTARTED';
 
-interface ProjectContributorIdentity {
+export interface ProjectContributorIdentity {
```

Comment:
ser ikke ut som den brukes andre plasser?
```suggestion
interface ProjectContributorIdentity {
```